docs/source/apps/mp-api.md

```diff
@@ -1,14 +1,19 @@
-# MarketPlace API
+# Standard API
 
-To declare which capabilities are supported by your application, a REST-API that complies with the MarketPlace specification must be set for your application.
-The API should be expressed using the [OpenAPI specification](https://www.openapis.org/) standard and be sent to the MarketPlace platform upon [registration](registration.md).
+The MarketPlace defines a standard API through which the registered application can be communicated and monitored via the MarketPlace.
+The API is defines a set of of endpoints or _capabilities_ to specify actions which MarketPlace applications may perform.
+An application provider may choose to support any subset of these capabilities, depending on the relevant functionalities.
 
-You can browse the official specification with redoc <a href="api.html" target="_blank">here</a>.
-The complete specification is hosted publicly on [GitHub](https://github.com/materials-marketplace/standard-app-api).
+The complete specification is hosted publicly on GitHub:<br>
+[https://github.com/materials-marketplace/standard-app-api](https://github.com/materials-marketplace/standard-app-api).<br>
+You can also browse the official specification with Redoc <a href="api.html" target="_blank">here</a>.
+
+Below is more information about the API specifications. [Here](../capabilities) more about app capabilities. Once completed, the application can be registered to the platform, as explained in [this](registration.md) page.
```

Comment:
```suggestion
Below is more information about the API specification. [Here](../capabilities), more about app capabilities. Once completed, the application can be registered on the platform, as explained in [this page](registration.md).
```

Reply:
Some changes I adopted and adapted.
Concerning the "register on", I think "to" is more accurate because that's what you are registering to, if that makes sense.

Reply:
I think after "to" you usually add a verb. It is meant for actions, I believe. But I'm not sure what would be the best option.

Reply:
Not always: "He goes to _school_".

Reply:
I mean in case of [register](https://ell.stackexchange.com/a/159339). But I don?t think _for_ is the answer either.

Reply:
I'm still in favor of "register to" :)